setup.py

```diff
@@ -27,7 +27,8 @@ def get_version():
         'numpy>=1.20.0',
         'opencv-python',
         'matplotlib',
-        'pyyaml'
+        'pyyaml',
+        'pillow'
```

Comment:
@onuralpszr do I have to specify `pillow` version here? or `pyproject` can handle this by itself?

Reply:
Setup.py is gone, I was working but I got fallen to slept last night, give me bit more time. I will handle it.  :)

Reply:
I know setup.py is gone. This was created long before that. Take your time.

Reply:
> @onuralpszr do I have to specify `pillow` version here? or `pyproject` can handle this by itself?

pyproject will handle it.